front_end/panels/timeline/CountersGraph.ts

```diff
@@ -127,18 +131,35 @@ export class CountersGraph extends UI.Widget.VBox {
     this._counterUI = [];
 
     this._countersByName = new Map();
-    this._countersByName.set(
-        'jsHeapSizeUsed',
-        this._createCounter(
-            i18nString(UIStrings.jsHeap), 'hsl(220, 90%, 43%)', Platform.NumberUtilities.bytesToString));
-    this._countersByName.set('documents', this._createCounter(i18nString(UIStrings.documents), 'hsl(0, 90%, 43%)'));
-    this._countersByName.set('nodes', this._createCounter(i18nString(UIStrings.nodes), 'hsl(120, 90%, 43%)'));
-    this._countersByName.set(
-        'jsEventListeners', this._createCounter(i18nString(UIStrings.listeners), 'hsl(38, 90%, 43%)'));
 
-    this._gpuMemoryCounter = this._createCounter(
-        i18nString(UIStrings.gpuMemory), 'hsl(300, 90%, 43%)', Platform.NumberUtilities.bytesToString);
-    this._countersByName.set('gpuMemoryUsedKB', this._gpuMemoryCounter);
+    this._countersByName.set('Coherent_LayerTextures', this._createCounter('Layer Textures', 'hsl(239, 100%, 50%)', CounterType.SimpleCounter));
+    this._countersByName.set('Coherent_ScratchTextures', this._createCounter('Scratch Textures', 'hsl(120, 90%, 43%)', CounterType.SimpleCounter));
+    this._countersByName.set('Coherent_SurfacesCounter', this._createCounter('Surface Textures', 'hsl(181, 90%, 43%)', CounterType.SimpleCounter));
+    this._countersByName.set('Coherent_ImagesCounter', this._createCounter('Images Textures', 'hsl(304, 900%, 50%)', CounterType.SimpleCounter));
+    this._countersByName.set('Coherent_RenoirFrameMemory', this._createCounter('Renoir Frame Memory', 'hsl(304, 900%, 50%)', CounterType.MemoryCounter, Platform.NumberUtilities.bytesToString));
+    this._gpuMemoryCounter = this._createCounter(UIStrings.gpuMemory, 'hsl(300, 90%, 43%)', CounterType.MemoryCounter, Platform.NumberUtilities.bytesToString);
+
+    this._countersByName.set('gpuMemoryUsed', this._gpuMemoryCounter);
+
+    this._showMemoryCounters = Common.Settings.Settings.instance().createSetting('timelineShowMemory', false);
+    this._showSimpleCounters = Common.Settings.Settings.instance().createSetting('timelineCounters', false);
+
+    this._showMemoryCounters.addChangeListener(this._onShowCountersChanged, this);
+    this._showSimpleCounters.addChangeListener(this._onShowCountersChanged, this);
+  }
+
+  _onShowCountersChanged(): void {
+
+    for (const counterUI of this._counterUI)
+    {
```

Comment:
In JS '{' are on the same line. Can you refactor them?
This includes functions, ifs, for...

Reply:
sure